superbench/config/amd_mi100_hpe.yaml

```diff
@@ -37,6 +37,7 @@ superbench:
   benchmarks:
     kernel-launch:
       <<: *default_local_mode
+      timeout: 120
```

Comment:
Besides, why only kernel-launch only? If we cannot make sure the setting, we can revise it in the next PR~~

Reply:
Just the PR description, I just take kernel-launch as example to show how to add timeout.

Reply:
I think we should not change the default on this.

Reply:
Your suggestion is that we don't change the config files in this PR?